setup.py

```diff
@@ -0,0 +1,13 @@
+from setuptools import setup
+
+setup(
+    name='xbundle',
+    version='0.1.0',
+    packages=['xbundle'],
+    scripts=['bin/xbundle_convert.py'],
```

Comment:
We can easily change the name of xbundle_convert.py to xbundle_convert without changing anything else. I just left it that way due to my auto pylint script (bad reason). I'll fix.